ArduPlane/quadplane.h

```diff
@@ -50,7 +50,7 @@ class QuadPlane
 
     // is VTOL available?
     bool available(void) const {
-        return initialised;
+        return initialised && enable;
```

Comment:
I was thinking about it more, and I think this change is probably wrong, both for performance and safety reasons, but I wanted to leave it in for the discussion.

Reply:
I agree, we shouldn't make that change